topk-js/src/expr/logical/comparable.rs

```diff
@@ -21,29 +21,24 @@ impl FromNapiValue for Comparable {
             return Ok(Comparable::Expr(expr.clone()));
         }
 
-        if let Ok(string) = String::from_napi_value(env, value) {
-            return Ok(Comparable::String(string));
+        let mut result: i32 = 0;
```

Comment:
instead of using the low-level napi apis, you can do something like:

```
match Value::from_napi_value(env, value)? {
    String(s) => Ok(Comparable::String(s)),
    v => Err(napi::Error::from_reason(format!("Unsupported comparable expression type: {}", v)))
}